arm-recoveryservices/2016-06-01/swagger/registeredidentities.json

```diff
@@ -0,0 +1,394 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2016-06-01",
+    "title": "RecoveryServicesClient",
+    "x-ms-code-generation-settings": {
+      "internalConstructors": false
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/certificates/{certificateName}": {
+      "put": {
+        "tags": [
+          "VaultCertificates"
+        ],
+        "description": "Upload a certificate for a resource.",
+        "operationId": "VaultCertificates_Upload",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/VaultName"

```

Comment:
Is the order of parameters intended? I have no clear expectations regarding this service, but it seems odd that `vaultName` is named prior to `resourceGroupName`, especially since this is the reverse order of how stuff appears in the path! Does this generate an intuitive method signature? (this comment may apply to other methods)